packages/compile-loader/index.js

```diff
@@ -1,12 +1,21 @@
 const babelMerge = require('babel-merge');
 
-module.exports = (neutrino, options = {}) => {
+module.exports = (neutrino, { ruleId = 'compile', useId = 'babel', ...options } = {}) => {
+  if (neutrino.config.module.rules.has(ruleId)) {
+    throw new Error(
+      '@neutrinojs/compile-loader has been used twice with the same `ruleId`.\n' +
+      'If you are including this preset manually to customise the compile rules\n' +
+      "configured by another preset, instead use that preset's own options to do so\n" +
+      '(such as the `babel` option when using the Neutrino web/react/node/... presets).'
```

Comment:
I'm confused by the double quote placement here…

Reply:
It avoids having to escape the single quote in the middle of the string (it's what https://prettier.io/playground/ outputs when using the "single quote" option and this string).